getting-started/modules/core/data-provider.md

```diff
@@ -0,0 +1,37 @@
+# Data provider
+
+The data provider module exposes any kind of data through an API. An engineer can expose data by writing a data
+provider. 
+
+This module was created for a specific use-case: the admin of the application should be able to configure a
```

Comment:
Not sure if I like this. Rather than say it's meant for a specific use case, maybe just use it as an example for why you might want to use this.